src/main/java/com/basho/riak/client/api/RiakClient.java

```diff
@@ -416,6 +417,18 @@ public static RiakClient newClient(Collection<HostAndPort> hosts, RiakNode.Build
         return cluster.shutdown();
     }
 
+
+    /**
+     * Implementation of the AutoCloseable-Inteface.
+     * <p>
+     *     The Client will be automatically shut down when used with try-with-resource statement
+     * </p>
+     */
+    @Override
+    public void close(){
+        this.shutdown();
```

Comment:
Shutdown method returns Future, therefore to respect AutoCloseable  it might be better to call as follows:

```
this.shutdown().get();
```